### PR TITLE
MULTIARCH-168 additional release notes

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/MULTIARCH-168?focusedCommentId=14026080&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14026080

### CODE REVIEW

**Comment by mburke5678** [comment]
@manojnkumar PTAL

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -1545,6 +1545,21 @@ $ systemctl reboot
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1784201[*BZ#1784201*])
 
+* Because of issues with `coreos-installer`, CoreOS cannot be installed on 
+bare-metal nodes that use NVMe drives with 4K sectors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1805249[*BZ#1805249*])
+
+* There is an issue with the `fw_enabled_large_send` setting on Power PC
```
Comment (manojnkumar) [inline release_notes/ocp-4-3-release-notes.adoc:1552]: Change: 
Power PC
to
IBM Power

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -1545,6 +1545,21 @@ $ systemctl reboot
...
 
+* Because of issues with `coreos-installer`, CoreOS cannot be installed on 
+bare-metal nodes that use NVMe drives with 4K sectors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1805249[*BZ#1805249*])
+
+* There is an issue with the `fw_enabled_large_send` setting on Power PC 
+systems that is causing VXLAN packet drop and can cause deployments to fail.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1816254[*BZ#1816254*])
+
+* For clusters on an IBM-Z infrastructure, due to
```
Comment (manojnkumar) [inline release_notes/ocp-4-3-release-notes.adoc:1556]: Change 
IBM-Z
to
IBM Power

**Comment by mburke5678** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/20658/files

**Pull request closed without merging**